primitive/primitive.go

```diff
@@ -51,6 +57,19 @@ type User struct { // type: 0x01
 	Master   *MasterKey    `json:"master"`
 }
 
+// User is the body of a user object
+type User struct { // type: 0x01
+	BaseUser
+	v2Schema
+	PublicKey *LoginPublicKey `json:"public_key"`
+}
+
+// UserV1 is the body of a user object
+type UserV1 struct { // type: 0x01
+	BaseUser
+	v1Schema
```

Comment:
To fit the style of the other structs put the v*Schema first, please.

And for now, include `mutable` on the User and UserV1 types, not the base one (again just to match the others)